Application/Transfer/Import/Standard/Mail/Service.php

```diff
@@ -0,0 +1,227 @@
+                 */
+                $Location = array(
+                    'Benutzername' => null,
+                    'Vorname' => null,
+                    'Nachname' => null
+                );
+                for ($RunX = 0; $RunX < $X; $RunX++) {
+                    $Value = trim($Document->getValue($Document->getCell($RunX, 0)));
+                    if (array_key_exists($Value, $Location)) {
+                        $Location[$Value] = $RunX;
+                    }
+                }
+
+                $countPersons = 0;
+                $countMissingPersons = 0;
+                $countDuplicatePersons = 0;
+                $countAccounts = 0;
+                $countMissingAccounts = 0;
+                $countAddMail = 0;
+
+                /**
+                 * Import
+                 */
+                if (!in_array(null, $Location, true)) {
+                    $error = array();
+                    for ($RunY = 1; $RunY < $Y; $RunY++) {
+                        $firstName = trim($Document->getValue($Document->getCell($Location['Vorname'], $RunY)));
+                        $lastName = trim($Document->getValue($Document->getCell($Location['Nachname'], $RunY)));
+                        $mail = trim($Document->getValue($Document->getCell($Location['Benutzername'], $RunY)));
+                        $mail = str_replace(' ', '', $mail);
+                        $addMail = false;
+                        $tblPerson = false;
+                        if ($firstName !== '' && $lastName !== '' && $mail != '') {
+                            if (($tblPersonList = Person::useService()->getPersonAllByFirstNameAndLastName($firstName, $lastName))) {
+                                if (count($tblPersonList) == 1) {
+                                    $countPersons++;
+                                    $addMail = true;
+                                } else {
+                                    $countDuplicatePersons++;
+                                    $error[] = 'Zeile: ' . ($RunY + 1) . ' Die Person ' . $firstName . ' ' . $lastName . ' wurde mehrmals gefunden';
+                                }
+                            } elseif (($tblPersonList = Person::useService()->getPersonAllByFirstNameAndLastName($this->refactorName($firstName), $this->refactorName($lastName)))) {
+                                if (count($tblPersonList) == 1) {
+                                    $countPersons++;
+                                    $addMail = true;
+                                } else {
+                                    $countDuplicatePersons++;
+                                    $error[] = 'Zeile: ' . ($RunY + 1) . ' Die Person ' . $firstName . ' ' . $lastName . ' wurde mehrmals gefunden';
+                                }
+                            } elseif (($tblPersonList = Person::useService()->getPersonListLikeFirstNameAndLastName($this->refactorName($firstName), $this->refactorName($lastName)))) {
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 180 characters; contains 184 characters](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)